src/error.lisp

```diff
@@ -223,13 +223,24 @@ NOTES and HELP-NOTES may optionally be supplied notes and help messages."
       ;; Print the error message and location
       (multiple-value-bind (line-number line-start-index)
           (get-line-from-index file-stream (coalton-error-location error))
```

Comment:
Should `get-line-from-index` be changed instead? Would the logic be equivalent?

Reply:
When I attempted to change `get-line-from-index` to match the expectation here, a number of other tests broke, so I think that may be something to address separately.

Reply:
Sounds good!